smartdashboard/pages/2_Database_Telemetry.py

```diff
@@ -0,0 +1,104 @@
+# BSD 2-Clause License
+#
+# Copyright (c) 2021-2023, Hewlett Packard Enterprise
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# 1. Redistributions of source code must retain the above copyright notice, this
+#    list of conditions and the following disclaimer.
+#
+# 2. Redistributions in binary form must reproduce the above copyright notice,
+#    this list of conditions and the following disclaimer in the documentation
+#    and/or other materials provided with the distribution.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+# FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+# OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+import argparse
+import os
+import pathlib
+import sys
+import time
+import typing as t
+
+import streamlit as st
+
+from smartdashboard.Experiment_Overview import get_parser
+from smartdashboard.utils.errors import SSDashboardError
+from smartdashboard.utils.ManifestReader import load_manifest
+from smartdashboard.utils.pageSetup import local_css, set_streamlit_page_config
+from smartdashboard.view_builders import db_telem_builder, error_builder
+
+
+def get_path(args: t.List[str]) -> str:
```

Comment:
let's pull manifest parsing out of any particular view since it's not tied specifically to DB telemetry. it might make better sense in a shared space (e.g. view.py, view_builders.py)

Reply:
move to ManifestReader as a static method

Reply:
In order to solve a circular import, I pulled out the `get_parser` function into its own file. The `execute` function in `Experiment_Overview` became unnecessary after moving `get_manifest_path` to `ManifestReader`, so it was removed..